test/workers/test_plan.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+import syft as sy
+
+import torch
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+def test_plan_built_locally():
+    @sy.func2plan()
+    def plan_abs(data):
+        return data.abs()
+
+    x = th.tensor([-1, 2, 3])
+    x_abs = plan_abs(x)
+
+    assert isinstance(plan_abs.__str__(), str)
+    assert len(plan_abs.readable_plan) > 0
+
+
+def test_plan_built_remotely(workers):
+    bob = workers["bob"]
+    alice = workers["alice"]
+
+    @sy.func2plan()
+    def plan_abs(data):
+        return data.abs()
+
+    plan_abs.send(bob)
+    x_ptr = th.tensor([-1, 7, 3]).send(bob)
+    p = plan_abs(x_ptr)
+    x_abs = p.get()
+
+    assert (x_abs == th.tensor([1, 7, 3])).all()
+
+    # Test send / get plan
+    plan_abs.get()
+    plan_abs.send(alice)
+
+    x_ptr = th.tensor([-1, 2, 3]).send(alice)
+    p = plan_abs(x_ptr)
+    x_abs = p.get()
+    assert (x_abs == th.tensor([1, 2, 3])).all()
+
+
+# def test_plan_remote_function(workers):
```

Comment:
Why are these commented out

Reply:
They are no longer valid behaviour we want to test

Reply:
I've removed them